src/config/types.ts

```diff
@@ -335,3 +349,25 @@ export type DateRangeType = {
   month: string;
   isFirstDay: boolean;
 };
+
+export interface FeatureInfoType {
+  bbox: number[];
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+}
+
+export interface requestFeatureInfo extends FeatureInfoType {
```

Comment:
PascalCase

Reply:
included in f37d4fe1544d6f98e61d8e03f1d27685fc2a0e2b